extensions/sequel-ace/src/database_list.tsx

```diff
@@ -39,7 +39,7 @@ export default function DatabaseList() {
       <List.EmptyView icon={{ source: "no-view.png" }} title="No databases found in Sequel Ace, go add one!" />
       {state &&
         state.grps?.map((item) => {
-          const subtitle = `${item.connections.length}` + ` item${item.connections.length > 1 ? "s" : ""}`;
+          const subtitle = `${item.connections.length}` + ` item${item.connections.length !== 1 ? "s" : ""}`;
```

Comment:
@nagauta I don’t see your comment in Github now, but I made this change so it would show “0 items” instead of “0 item”

Reply:
@macbookandrew 
Ah, I see, zero is plural.
I understood, thanks!